kontena-develop.yml

```diff
@@ -27,7 +27,7 @@ services:
     image: ludx/particl:0.16
     instances: 1
     volumes:
-      - /root/data/particld-dev2:/root/.particl
+      - /root/data/particld-dev2:/root/.particl:cached
```

Comment:
this is the kontena stack file, adding cached here doesnt work and doesnt make sense

Reply:
remove cached